components/post_view/post_add_channel_member/post_add_channel_member.jsx

```diff
@@ -66,19 +65,17 @@ export default class PostAddChannelMember extends React.PureComponent {
     }
 
     handleAddChannelMember = () => {
-        const {currentUser, team, channel, postId, userIds, usernames} = this.props;
+        const {currentUser, postId, userIds, usernames} = this.props;
         const post = this.props.actions.getPost(getState(), postId) || {};
 
-        if (post.channel_id === channel.id) {
+        if (post && post.channel_id) {
             userIds.forEach((userId, index) => {
-                this.props.actions.addChannelMember(channel.id, userId);
-                sendAddToChannelEphemeralPost(currentUser, usernames[index], channel.id, post.root_id);
+                this.props.actions.addChannelMember(post.channel_id, userId);
+                sendAddToChannelEphemeralPost(currentUser, usernames[index], post.channel_id, post.root_id);
             });
 
             this.props.actions.removePost(post);
         }
-
-        browserHistory.push(`/${team.name}/channels/${channel.name}`);
```

Comment:
There's no point to redirect to the channel where at-mentioned user is added, so user who adds a user will remain on his currently viewing channel.

Reply:
After removing that line team property is not longer needed. and from the channel, is only needed the channelType, maybe make sense to only receive the channelType.

Reply:
Make sense, I forgot to clean it up. Thanks!